README.md

```diff
@@ -141,6 +141,14 @@ The yarn build scripts default to production for libraries if `PANOPTES_ENV` is
 - `zooniverse/fe-content-pages`: Built from the Dockerfile in `packages/app-content-pages`. Runs `yarn build` and `yarn start` in `packages/app-content-pages` from `zooniverse/front-end-monorepo:latest`.
 - `zooniverse/fe-project`: Built from the Dockerfile in `packages/app-project`. Runs `yarn build` and `yarn start` in `packages/app-project` from `zooniverse/front-end-monorepo:latest`.
 
+### Publishing
+When publishing an individual package to [npm](https://www.npmjs.com/), first cd into the repo you would like to deploy (within the packages folder), then:
+1. Update changelog and commit
+1. `yarn version --major --no-git-tag-version`
```

Comment:
It won't always be a major version bump, so note here to use the correct semver for the version being released.

Reply:
Good catch